apps/src/gamelab/spritelab/spriteUtils.js

```diff
@@ -0,0 +1,222 @@
+var spriteId = 0;
+var nativeSpriteMap = {};
+var inputEvents = [];
+var behaviors = [];
+
+export var background;
+export var title = '';
+export var subtitle = '';
+
+export function reset() {
+  spriteId = 0;
+  nativeSpriteMap = {};
+  inputEvents = [];
+  behaviors = [];
+  background = 'white';
+  title = subtitle = '';
+}
+
+/**
+ * Returns a list of all sprites that have the specified animation.
+ * Called on each tick of the draw loop because animations can change throughout runtime.
+ * @param {string} animation - animation name
+ */
+function allSpritesWithAnimation(animation) {
+  let group = [];
+  Object.keys(nativeSpriteMap).forEach(spriteId => {
+    if (nativeSpriteMap[spriteId].getAnimationLabel() === animation) {
```

Comment:
If I call delete on a spriteId, and then call allSpritesWithAnimation, will it throw an error "Cannot read property getAnimationLabel of undefined"?

Reply:
Oh wait. Nevermind.